src/web/components/ShareIcons.tsx

```diff
@@ -109,7 +117,9 @@ export const ShareIcons: React.FC<{
 			{displayIcons.includes('email') && (
 				<li className={liStyles} key="email">
 					<a
-						href={`mailto:?subject=${webTitle}&body=https://www.theguardian.com/${pageId}&CMP=share_btn_link`}
+						href={`mailto:?subject=${webTitle}&body=${encodeUrl(
```

Comment:
`encodeTitle` here?

Reply:
Seconded

Reply:
I've added encoding for the subject line of emails as well. I tested this locally and it worked before, but also works after and I think the added security of encoding is definately what we want